app/components/hopper-frontend.js

```diff
@@ -2,9 +2,18 @@ import Ember from 'ember';
 
 export default Ember.Component.extend({
     isElementDrawerOpen: false,
+    isTitleEditing: false,
```

Comment:
Correct spelling for attribute name is `isTitleEdited`. Please change the attriibute name.